### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by igordavydsson**
Update README.
Make docker container more verbose for dev purposes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,10 +67,21 @@ If you experience a missing locale error, you can try configuring the locale:
 ```
 python manage.py migrate
 python manage.py createsuperuser
-python manage.py runserver
+python manage.py runserver 8000
```
Comment (tuomas777) [inline README.md]: Not sure why this change was needed, isn't localhost:8000 the default? :)

Comment (igordavydsson) [reply README.md]: oh, yeah, wasn't sure about that, but checked django's course and you are totally right 👍
